sylvia-derive/Cargo.toml

```diff
@@ -12,18 +12,19 @@ categories = ["api-bindings", "wasm"]
 
 [features]
 mt = []
+library = []
```

Comment:
This is not a valid feature name. Sylvia is always used as a library. Rename to `entry_points`.

Reply:
because as far as I'm concerned we cannot disable features and also I think by default entry_points should be disabled I will change it to `no_entry_points` and users can define on their side some feature f.e. `library` that if set would also set `no_entry_points` for sylvia

Reply:
Ya, this is also somehow related to the recent discussion to change it the other way... but doing it as `no_entry_points` is completely fine.

However, I am pretty sure changing it to `entry_points`, and then adding the reversed logic - adding the `binary` feature and enabling it only on the contracts being built as binaries should solve the problem. Unfortunately I am not sure how it would work with workspace optimizer :/

Reply:
We could do that but I think from user perspective it would be weird to add "entry_points" flag to disable them. That's why I opt for the less nice but making more sense "no_entry_points"